src/Core/WinSWCore/Configuration/ServiceAccount.cs

```diff
@@ -0,0 +1,22 @@
+﻿using YamlDotNet.Serialization;
+
+namespace winsw.Configuration
+{
+    public class ServiceAccount
+    {
+        [YamlMember(Alias = "name")]
```

Comment:
What's this?

Reply:
I have misinterpret user and name. I updated this.